EIPS/eip-1.md

```diff
@@ -207,6 +207,10 @@ EIPs may also have a `superseded-by` header indicating that an EIP has been rend
 
 References to other EIPs should follow the format `EIP-N` where `N` is the EIP number you are referring to.  Each EIP that is referenced in an EIP **MUST** be accompanied by a relative markdown link the first time it is referenced, and **MAY** be accompanied by a link on subsequent references.  The link **MUST** always be done via relative paths so that the links work in this GitHub repository, forks of this repository, the main EIPs site, mirrors of the main EIP site, etc.  For example, you would link to this EIP with `[EIP-1](./eip-1.md)`.
 
+## Linking to outside sources
+
+EIPs **MUST NOT** link to external domains.
```

Comment:
```suggestion
EIPs should link only to credible external domains at addresses that are expected to continue working for the foreseeable future. Ephemeral sources such as social media, chat logs, and comments should be disallowed. Discretion is left up to EIP Editors.
```

Reply:
I would be okay with something along these lines.

Reply:
In my experience, "discretion left up to editors" is read as "please argue with the editors on why Peepeth is not an ephemeral source".  As I mentioned above, my primary concern is that any "fuzziness" is going to eat precious editor time.  Whether it is the editors reviewing every external link, or it is editors resolving disputes about what is "credible" or what is "ephemeral" doesn't really change the time sink much, just changes the mechanism.

Reply:
If we allow *some* links we either need to build a comprehensive list of valid sources and then maintain it or we need to allow editors to decide if the link is credible. I don't have a strong opinion between the two, only that I don't want to maintain a list right now.

Reply:
I also don't want to maintain a list.  😁  I think I would prefer something like:
```suggestion
EIPs should avoid any external links.  Some exceptions have been made by editors in the past such as linking to the `eth2.0-specs` repository or linking to an RFC at ietf.org in certain situations, but such exceptions are relatively rare and avoiding external links is the best strategy for getting your EIP merged.
```

Reply:
This is also fine to me.